granary/test/test_flickr.py

```diff
@@ -0,0 +1,651 @@
+"""Unit tests for flickr.py
```

Comment:
yay thanks for the tests and test data! they look great. maybe also consider testing `get_comment()`; otherwise this looks good as is.